<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupForm</class>
 <widget class="QWidget" name="SetupForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="selectWidget" native="true">
     <property name="styleSheet">
      <string notr="true">border-color: rgb(118, 118, 118);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>11</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="lbGroup">
           <property name="text">
            <string>&amp;Group:</string>
           </property>
           <property name="buddy">
            <cstring>leftListWidget</cstring>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QListWidget" name="leftListWidget"/>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QPushButton" name="btnSelectOption">
               <property name="text">
                <string>&gt;&gt;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnRemoveItem">
               <property name="text">
                <string>&lt;&lt;</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QListWidget" name="rightListWidget"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="lbCourse">
          <property name="text">
           <string>&amp;Course:</string>
          </property>
          <property name="buddy">
           <cstring>comboCourse</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboCourse"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="lbSemester">
          <property name="text">
           <string>&amp;Semester:</string>
          </property>
          <property name="buddy">
           <cstring>comboSemester</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboSemester"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnOkSetup">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboCourse</tabstop>
  <tabstop>comboSemester</tabstop>
  <tabstop>btnOkSetup</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
